### PR TITLE
Issue #22: Implemented vulnerabilities tab and redesigned the app tabs

### LINKED ISSUE #22: Implement Vulnerabilities tab and redesign tabs

### PR DESCRIPTION
**Pull request by thedarkspy**
Closes #22

### CODE REVIEW

File: web/src/components/Contract/Tabs/Reports/Reports.js [hunk 1]
```diff
@@ -1,60 +1,51 @@
...
+            text = report.data.split('.').join('\n').trim();
+            break;
+          default:
+            if (report.data) text = report.data.trim();
+        }
+      } else {
+        text = 'The contract is not supported by this scanner.';
       }
+    } else {
+      text = 'The contract was not scanned by this scanner yet.';
```
Comment (boshmaf) [inline web/src/components/Contract/Tabs/Reports/Reports.js:48]: `The contract has not been analyzed by this scanner yet.`

Grammer + different verb.

File: web/src/components/Contract/Tabs/Vulnerabilities/Vulnerabilities-styles.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+export const stylesCreator = (theme) => ({
+  Default: {
+    root: {
+      height: 454,
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+    tableContainer: {
+      padding: 16,
```
Comment (boshmaf) [inline web/src/components/Contract/Tabs/Vulnerabilities/Vulnerabilities-styles.js:16]: Please use theme.spacing instead for paddings and margins.

File: web/src/components/Contract/Tabs/Vulnerabilities/Vulnerabilities.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+                <TableCell>{row.severity}</TableCell>
+                <TableCell>{row.scannedBy}</TableCell>
+                <TableCell>{row.version}</TableCell>
+                <TableCell>{row.timestamp}</TableCell>
+              </TableRow>
+            );
+          })}
+        </TableBody>
+      </Table>
+    </TableContainer>
```
Comment (boshmaf) [inline web/src/components/Contract/Tabs/Vulnerabilities/Vulnerabilities.js:97]: Please use [DataTableRaw](https://github.com/cibr-qcri/kansa/tree/main/web/src/components/DataTableRaw) component instead. Also, please make sure you're using this *for all Tabs that show data tables.* This raw component handles pagination, empty results, etc.

**Review by boshmaf — CHANGES_REQUESTED** [review]

File: web/src/components/Contract/Tabs/Vulnerabilities/Vulnerabilities-styles.js (hunk 2 above)
Comment (thedarkspy) [reply web/src/components/Contract/Tabs/Vulnerabilities/Vulnerabilities-styles.js]: Addressed

**Review by boshmaf — APPROVED** [review]

**Pull request merged**